test/e2e/performanceprofile/functests/8_performance_workloadhints/workloadhints.go

```diff
@@ -617,14 +617,34 @@ var _ = Describe("[rfe_id:49062][workloadHints] Telco friendly workload specific
 					HighPowerConsumption:  pointer.Bool(true),
 					RealTime:              pointer.Bool(true),
 				}
-				EventuallyWithOffset(1, func() string {
-					err := testclient.ControlPlaneClient.Update(context.TODO(), profile)
-					if err != nil {
-						statusErr, _ := err.(*errors.StatusError)
-						return statusErr.Status().Message
-					}
-					return fmt.Sprint("Profile applied successfully")
-				}, time.Minute, 5*time.Second).Should(ContainSubstring("HighPowerConsumption and PerPodPowerManagement can not be both enabled"))
+				if hypershift.IsHypershiftCluster() {
+					hostedClusterName, err := hypershift.GetHostedClusterName()
+					np, err := nodepools.GetByClusterName(ctx, testclient.ControlPlaneClient, hostedClusterName)
+					Expect(err).ToNot(HaveOccurred())
+					EventuallyWithOffset(1, func() string {
+						err := testclient.ControlPlaneClient.Update(context.TODO(), profile)
+						if err == nil {
+							reason := "ValidationFailed"
+							messages, err := nodepools.NodePoolStatusMessages(ctx, testclient.ControlPlaneClient, np.Name, np.Namespace, reason)
+							if err != nil {
+								statusErr, _ := err.(*errors.StatusError)
+								return statusErr.Status().Message
+							}
+							return messages[0]
+						}
+						return ""
+					}, time.Minute, 5*time.Second).Should(ContainSubstring("HighPowerConsumption and PerPodPowerManagement can not be both enabled"))
+				} else {
+
+					EventuallyWithOffset(1, func() string {
+						err := testclient.ControlPlaneClient.Update(context.TODO(), profile)
```

Comment:
ditto

Reply:
In the normal openshift it doesn't allow you to update the performance profile at all hence it's in the Eventually WithOffset loop.

Reply:
I see, this is because there's a mutation webhook protecting the code so it will block the request from reaching the API server. 
I'm pretty we're testing this functionality elsewhere.